lib/ansiblelint/utils.py

```diff
@@ -573,3 +575,82 @@ def construct_mapping(node, deep=False):
     except (yaml.parser.ParserError, yaml.scanner.ScannerError) as e:
         raise SystemExit("Failed to parse YAML in %s: %s" % (filename, str(e)))
     return data
+
+
+def add_skipped_rules(orig_file_text, file_type):
+    yaml = ruamel.yaml.YAML()
+    data = yaml.load(orig_file_text)
```

Comment:
load or load`s`?

Reply:
Also, should we use safe_load?

Reply:
`ruamel.yaml` uses `load` instead of `loads`

We are using `yaml = ruamel.yaml.YAML()` and their [docs](https://yaml.readthedocs.io/en/latest/dumpcls.html#dumping-python-classes) state:
> Only yaml = YAML(typ='unsafe') loads and dumps Python objects out-of-the-box. And since it loads any Python object, this can be unsafe.